esi_leap/common/policy.py

```diff
@@ -29,6 +29,15 @@
     policy.RuleDefault('is_lessee',
                        'role:lessee or role:esi_leap_lessee',
                        description='Lessee API access'),
+    policy.RuleDefault('is_offer_owner',
+                       'project_id:%(offer.project_id)s',
+                       description='Owner of offer'),
```

Comment:
Is there a reason why there's no `'is_offer_lessee`? I do know lessees can be specified when creating an offer, does this not factor into any policy decisions?

Reply:
Ah, yes - it's because I'm not certain how easy it is to create a policy rule that says "if %(offer.lessee_id) is None or project_id:%(offer.project_id)s"

Reply:
Now that I think about it - I was going to look into this anyway, but then I remembered: the offer lessee_id actually represent an entire project tree. Oslo policy has relatively simple matching rules that don't accommodate for that.